doc/devices/qpu.rst

```diff
@@ -64,33 +62,18 @@ These operations include:
     pennylane_forest.ops.ISWAP
     pennylane_forest.ops.PSWAP
```

Comment:
The three gates mentioned here are now supported in PennyLane. Maybe remove the last part of this section?